Uni_life_tracker/app/src/main/res/layout/activity_expenses_landing.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
align the whole activity with LinearLayout (rather than constraint)

Reply:
fixed